packages/legend-studio/style/components/editor/mapping-editor/_filter-panel.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use '../../../mixins' as *;
+
+.class-mapping-filter-editor {
+  &__content {
+    height: 100%;
+    width: 100%;
+  }
+
+  &__element__lambda-editor {
```

Comment:
I don't think you need to do this, for `LambdaEditor`, I thought a combination of `forceExpansion` and `useBaseEditorSettings` would make this work instead of hard-coding the height like this